package/setup.py

```diff
@@ -494,11 +494,12 @@ def dynamic_author_list():
           classifiers=CLASSIFIERS,
           cmdclass=cmdclass,
           requires=['numpy (>=1.10.4)', 'biopython', 'mmtf (>=1.0.0)',
-                    'networkx (>=1.0)', 'GridDataFormats (>=0.3.2)', 'joblib'],
+                    'networkx (>=1.0)', 'GridDataFormats (>=0.3.2)', 'joblib',
+                    'scipy', 'matplotlib (>=1.5.1)'],
```

Comment:
We still have our own hausdorff distance I think. You can open an issue for switching to the scipy implementation with a fallback to our own.